src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -501,4 +502,11 @@ public ResponseEntity deleteAttachment(
             );
         }
     }
+
+    @GetMapping("/eligibility")
```

Comment:
Spec needs updating to align to this new end-point name and what's returned. @SVarallo might be doing this? Strictly speaking, should be done before this code is merged.

Reply:
PR for this at: https://github.com/companieshouse/strike-off-objections-api/pull/77